pkgs/development/tools/makerpm/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, zlib, libarchive, openssl }: 
+
+stdenv.mkDerivation { 
+  name = "makerpm-1.0";
+  version = "1.0";
+
+  builder = ./builder.sh;
+
+  buildInputs = [ zlib libarchive openssl ];
+
+  src = fetchFromGitHub {
+    owner = "ivan-tkatchev";
+    repo = "makerpm";
+    rev = "141a4444b9bcb1df27ee33c4e0855ab58128017e";
+    sha256 = "1q76zv0qs5gx9w0qfd64kldw9gngafr16izh0prvw4cd8kn4cqrk";
+  };
+
+  meta = {
+    homepage = https://github.com/ivan-tkatchev/makerpm/;
+    description = "A clean, simple RPM packager reimplemented completely from scratch";
+    license = stdenv.lib.licenses.free;
+    platforms = stdenv.lib.platforms.all;
+    maintainers = [ "tkatchev@gmail.com" ];
```

Comment:
You should add an attribute to [lib/maintainers.nix](https://github.com/NixOS/nixpkgs/blob/master/lib/maintainers.nix) and use `stdenv.lib.maintainers` here.
The convention is to use your github username a the attribute name.